src/components/GenderViolenceMain/index.jsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import { Container, Row, Col } from "react-bootstrap";
+import { Link } from "react-router-dom";
 import stopGenderViolenceImg from "../../images/home/stop-gender-violence.svg";
```

Comment:
we are using camelCase in naming images

Reply:
Jamil I didn't import this image. I just added link to button :)

Reply:
I would say that it doesn't matter how svg file is called if it's imported in camelCase